teamsupport/models.py

```diff
@@ -58,15 +66,16 @@ def create(cls, user_email, user_first_name, user_last_name, title,
                 LastName=user_last_name
             )
 
+        ticket_type_id = cls._get_ticket_type_id(config.DEFAULT_TICKET_TYPE)
+
         data = {
             'Name': title,
+            'ContactID': contact.id,
             'TicketStatusID': cls._get_ticket_status_id(
-                config.DEFAULT_TICKET_STATUS),
-            'TicketTypeID': cls._get_ticket_type_id(
-                config.DEFAULT_TICKET_TYPE),
-            'ContactID': contact.id
+                ticket_type_id, config.DEFAULT_TICKET_STATUS),
+            'TicketTypeID': ticket_type_id,
+            'IsVisibleOnPortal': True,
         }
-        data.update(params)
```

Comment:
Why was this removed? If it is being removed shouldn't the `**param` parameter also be removed?